api/client/proto/authservice.proto

```diff
@@ -1389,6 +1410,11 @@ message CreateAuthenticateChallengeRequest {
         // VerifyAccountRecovery (step 2 of the recovery process after RPC StartAccountRecovery).
         string RecoveryStartTokenID = 2
             [ (gogoproto.jsontag) = "recovery_start_token_id,omitempty" ];
+        // Issues a challenge for the currently-authenticated user.
```

Comment:
Nit: Not a huge deal but since these turn into Go fields, these should be Godoc-style comments e.g. "ContextUser is...", "Passwordless is ...", etc throughout this proto file.

Reply:
Done.

Reply:
Because now both messages and fields under CreateAuthenticateChallengeRequest use the same naming convention (UpperCamel), it created a naming conflict between the ContextUser/Passwordless message/fields. To solve it I moved the ContextUser and Passwordless inner messages to the top-level, which seems to be a reasonable solution. (We could also rename the message in the future without any compatibility side-effects.)

Let me know if you have other suggestions.